### ISSUE TITLE
formID - initiative 10-10EZ - Application status

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#95313]
## Benefit to the Veteran
As a Veteran applying for VA healthcare, I want to be able to log into VA.gov and easily view the current status of my healthcare application, which can be 'In Process,' 'Enrolled,' 'Declined,' or 'Saved In-Progess,' and access guidance on what (if anything) to do next based on my application's status

## Hypothesis or Bet
**If** _we make this change_ **then** _we expect this to happen_.

## OKR
_Which Objective / Key Result does this epic push forward?_

## Background
we want to let the Veteran know if they have a saved (not yet submitted) form, a submitted but in-progress review, enrolled or declined (we've talked about looking at other statuses too, which can be future iterations) - Not only on the Application intro page, but also in MyVA or other appropriate [VA.gov](http://va.gov/) location.

- _What details are necessary for understanding the specific work or request tracked by this issue?_
- _What are we doing within the scope of this ticket?_
- _Is there anything specifically out of scope of this ticket?_
- _Is there anything that must be done before this ticket? (link ticket as dependency)_
- _Are there any external dependencies?_

## Resources
- Product or Initiative brief links 
- Design links
- Screenshots
- Reference docs/tickets where we have this kind of work documented
- Etc.
---
## Acceptance Criteria/Definition of Done
-  _What will be created or happen as a result of this story?_

---

```[tasklist]
### Tasks
- [ ] Research
- [ ] Design
- [ ] Team review & signoff
- [ ] UI update
- [ ] Team review & signoff
- [ ] BE update
- [ ] QA
- [ ] Team review & signoff
- [ ] End to end testing
- [ ] Release Plan
- [ ] Cypress test suite
- [ ] Team review & signoff
- [ ] Launch to Prod
- [ ] Update Source of Truth
```

---
## How to configure this issue
- [ ] **Labeled with Team** (`1010-team`)
- [ ] **Labeled with Form** (`1010-ez`, `1010-ezr`, `1010-cg`)
- [ ] **Labeled with Project** (`facilities-cg`, `registration`,  `toxic-exposure`, etc.)

**Comment by hdjustice** [comment]
Pending;Other - Less than 20 records at any given time
 
Pending;Unverified - this status is temporary but the totals hover around 150k
 
Pending;MT - Current count is 105k

**Comment by hdjustice** [comment]
Met with Mitch on this one.
There's been difficulty in pinpointing the required attributes/statuses that MHV/My-Health relies on to allow access to the my-health.va.gov space, which inhibits our confidence on whether we can direct Veterans to the EZR when more information is needed (such as financials, military documentation).

Next steps: At this point, we can consider this exploration closed and will look into a development spike to test out use cases on users with different attributes/statuses. Keeping in mind the reduction and restructure of the team, this will be prioritized after Confirmation flow/V2 and Military Service info work.
